google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -685,7 +687,7 @@ def public_url protocol: :https
         #   shared_url = file.signed_url method: "GET",
         #                                headers: {
         #                                  "x-goog-acl" => "public-read",
-        #                                  "x-goog-meta-foo" => bar,baz"
+        #                                  "x-goog-meta-foo" => "bar,baz"
```

Comment:
Concerning that this typo was passing doctest, but I see that `signed_url` is skipped in doctest with this comment:

> Due to failing line in example: key = OpenSSL::PKey::RSA.new "-----BEGIN PRIVATE KEY-----\n..."